app/graphql/resolvers/base.rb

```diff
@@ -0,0 +1,9 @@
+module Resolvers
+  class Base < GraphQL::Schema::Resolver
+    argument_class Types::BaseArgument
+    
+    def current_user
+      @context[:current_user_id]
```

Comment:
this method always returns `nil`, since you are not putting `current_user_id` anywhere into the context. You only put `current_user` in graphql_controller